src/lint-changed.ts

```diff
@@ -45,59 +45,89 @@ const getLastTag = () => git("describe --tags --abbrev=0 HEAD^");
 const getChangedFiles = (event: string) =>
   git(`diff --name-only ${event}`).then(r => r.split("\n").filter(n => !!n));
 
+/**
+ * Checks for files that have changed since the last tag on the master branch
+ */
+const checkMasterForChangedFiles = async () => {
+  const [tagFetchError, lastTag] = await to(getLastTag());
```

Comment:
Assuming that highter up the chain handles the `catch`, yah?

Reply:
The `to` function is catching any exceptions and making sure a promise is always returned.